styles.css

```diff
@@ -51,6 +61,12 @@ body {
   height: 150px;
   width: 150px;
   padding: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.child-3 {
+  margin-top: auto;
 }
```

Comment:
To solve this using flexbox you can use the property `flex-grow` for the `child-2` to let him take all the available free space.

```
.child-2 {
  flex-grow: 1;
}
```